src/test/java/ua/yakaboo/WebDriverSettings.java

```diff
@@ -0,0 +1,21 @@
+package ua.yakaboo;
+
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.*;
+
+public class WebDriverSettings {
+
+    public ChromeDriver driver;
+
+    @BeforeMethod
+    public void setUp() {
+        System.setProperty("webdriver.chrome.driver", "src/main/resources/drivers/chromedriver87");
+        driver = new ChromeDriver();
```

Comment:
Use WebDriver manager for driver creation

Reply:
Done